services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaControllerITest.kt

```diff
@@ -157,6 +160,70 @@ class HankeKayttajaControllerITest(@Autowired override val mockMvc: MockMvc) : C
         }
     }
 
+    @Nested
+    inner class WhoAmIByHanke {
+        private val url = "/hankkeet/my-permissions"
+
+        private val tunnus1 = "HAI23-1"
+        private val kayttaja1 = randomUUID()
+
+        private val tunnus2 = "HAI23-2"
+
+        private val hankePermissions =
+            listOf(
+                HankePermission(
+                    hankeTunnus = tunnus1,
+                    hankeKayttajaId = kayttaja1,
+                    kayttooikeustaso = Kayttooikeustaso.KAIKKI_OIKEUDET,
+                    permissionCode = 1152921504606846975,
```

Comment:
To avoid the magic value:
```suggestion
                    permissionCode = PermissionCode.entries.sumOf { it.code },
```